airflow/jobs.py

```diff
@@ -94,25 +94,54 @@ class BaseJob(Base, LoggingMixin):
         Index('idx_job_state_heartbeat', state, latest_heartbeat),
     )
 
+    heartrate = conf.getfloat('scheduler', 'JOB_HEARTBEAT_SEC'),
```

Comment:
DEAR GOD. That took like 4 hours to get enough of a working set up to get the log for!